index.js

```diff
@@ -29,60 +23,69 @@ function readAssetsJson(path) {
-    constructor({ development = false, path = './assets.json' }) {
+    constructor({ js, css, development = false, path = './assets.json' }) {
         let meta = readAssetsJson(path);
+
         const {
             server,
-            inputs,
+            js: { input: jsInput, options: jsOptions },
+            css: { input: cssInput, options: cssOptions },
             organisation,
             name,
-            version,
-            tagOptions = {},
+            version
         } = meta;
         this.scripts = [];
         this.styles = [];
 
-        if (development && meta.development) {
-            if (meta.development.js) {
+        if (development) {
+            if (js) {
                 let script = {};
-                if (typeof meta.development.js !== 'string') {
-                    script = new AssetJs(meta.development.js);
+                if (typeof js !== 'string') {
+                    script = new AssetJs(js);
                 } else {
                     script = new AssetJs({
                         type: 'module',
-                        value: meta.development.js,
+                        value: js,
+                        ...jsOptions
                     });
                 }
 
                 this.scripts.push(script);
             }
-            if (meta.development.css) {
+            if (css) {
                 let style = {};
-                if (typeof meta.development.css !== 'string') {
-                    style = new AssetCss(meta.development.css);
+                if (typeof css !== 'string') {
+                    style = new AssetCss(css);
                 } else {
-                    style = new AssetCss({ value: meta.development.css });
+                    style = new AssetCss({ value: css, ...cssOptions });
                 }
 
                 this.styles.push(style);
             }
             return;
         }
 
-        if (inputs.js) {
+        if (jsInput) {
             this.scripts.push(
                 new AssetJs({
-                    ...tagOptions.js,
-                    value: `${server}/${organisation}/js/${name}/${version}/index.js`,
                     type: 'module',
+                    ...jsOptions,
+                    value: `${server}/${organisation}/pkg/${name}/${version}/main/index.js`
+                })
+            );
+            this.scripts.push(
+                new AssetJs({
+                    ...jsOptions,
+                    type: 'iife',
+                    value: `${server}/${organisation}/pkg/${name}/${version}/ie11/index.js`
                 })
             );
         }
-        if (inputs.css) {
+        if (cssInput) {
             this.styles.push(
                 new AssetCss({
-                    ...tagOptions.css,
-                    value: `${server}/${organisation}/css/${name}/${version}/index.css`,
+                    ...cssOptions,
+                    value: `${server}/${organisation}/pkg/${name}/${version}/main/index.css`
```

Comment:
In core I've made some helper functions to construct these, should they go into a common module for reuse perhaps?

Reply:
was wondering about that, worth it I think for sure